manifests/rabbitmq/config.pp

```diff
@@ -16,88 +16,92 @@
 
   $ssl_dir = "${sensu::etc_dir}/ssl"
 
-  if $sensu::rabbitmq_ssl_cert_chain or $sensu::rabbitmq_ssl_private_key {
-    file { $ssl_dir:
-      ensure  => directory,
-      owner   => $sensu::user,
-      group   => $sensu::group,
-      mode    => '0755',
-      require => Package['sensu'],
-    }
-
-    # if provided a cert chain, and its a puppet:// URI, source file form the
-    # the URI provided
-    if $sensu::rabbitmq_ssl_cert_chain and $sensu::rabbitmq_ssl_cert_chain =~ /^puppet:\/\// {
-      file { "${ssl_dir}/cert.pem":
-        ensure  => file,
-        source  => $sensu::rabbitmq_ssl_cert_chain,
+  if $sensu::rabbitmq_ssl == false {
+    # do nothing
+  } else {
```

Comment:
You would still need to wrap every reference to rabbitmq_ssl_* in this class with with if rabbitmq_ssl = false do nothing, or change the default behavior of the module.